ProQuest.js

```diff
@@ -194,14 +200,26 @@ function doWeb(doc, url, noFollow) {
 				articles.push(item);
 			}
 			
-			if (articles[0].indexOf("ebraryresults") > -1) {
-				// if the first result is for ebrary, the rest are also ebrary
-				ZU.processDocuments(articles, function(doc) {
-					var translator = Zotero.loadTranslator("web");
-					translator.setTranslator("2abe2519-2f0a-48c0-ad3a-b87b9c059459");
-					translator.setDocument(doc);
-					translator.translate();
-				});
+			if (isEbrary) {
+				if(Zotero.isBookmarklet) {
+					// The bookmarklet can't use the ebrary translator
+					
+					var urlKeys = Object.keys(items);
+					var refs = [];
+					
+					for(var i = 0; i < urlKeys.length; i++) {
```

Comment:
`for (var i in items)` for iterating objects (not arrays), no need to get the keys beforehand.